assets/scripts/app/__tests__/PrintContainer.test.js

```diff
@@ -2,6 +2,7 @@
 import React from 'react'
 import { renderWithRedux } from '../../../../test/helpers/render'
 import PrintContainer from '../PrintContainer'
+import { screen } from '@testing-library/react'
```

Comment:
Let's move this line to just under `import React`, as such:

```js
import React from 'react'
import { screen } from '@testing-library/react'
```

Generally, we want to import libraries from packages first, then scripts. See [import order](https://streetmix.readthedocs.io/en/latest/contributing/code/styleguide/#import-order) guidelines.